client/js/views/GameView.vue

```diff
@@ -393,6 +358,14 @@ export default {
     };
   },
   computed: {
+    ...mapGetters([
+      'player',
+      'playerPointTotal',
+      'playerQueenCount',
+      'opponent',
+      'opponentPointTotal',
+      'opponentQueenCount',
+    ]),
```

Comment:
With the move of a lot of the shared data to the store, it's as simple as mapping the data each component needs now which is a nice change imo.